docs/stdlib/set.rst

```diff
@@ -425,14 +432,15 @@ Sets
 
     :index: cardinality existence empty
 
-    Check that the input set contains at least one element.
+    Returns :eql:type:`anytype` based on condition of if the set contains a
+    value.
```

Comment:
```suggestion
    Checks that the input set contains at least one element.
```

Reply:
```suggestion
    Returns ``true`` if the given set contains at least one element.
```

Reply:
I like @raddevon 's suggestion, I prefer using "returns" to explain that this is the resulting return value/type we have.